packages/components/Grid/src/Grid.tsx

```
@@ -0,0 +1,129 @@
+    justify,
+    alignItems,
+    css,
+    ...otherProps
+  } = props;
+
+  const getResponsiveColumns = (cols: GridProps["columns"]) => {
+    if (typeof cols === "number") return `repeat(${cols}, 1fr)`;
+
+    return Object.entries(cols).reduce(
+      (acc, [breakpoint, value]) => ({
+        ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)